util/build-gnu.sh

```diff
@@ -355,3 +355,8 @@ test \$n_stat1 -ge \$n_stat2 \\' tests/ls/stat-free-color.sh
 # * the selinux crate is handling errors
 # * the test says "maybe we should not fail when no context available"
 "${SED}" -i -e "s|returns_ 1||g" tests/cp/no-ctx.sh
+
+# Create a file not owned by current user for fail-eperm test
+if command -v sudo >/dev/null 2>&1; then
```

Comment:
please add a comment explaining the why :)

Reply:
Whoops, sorry that this is a common review. Added more description but will work on adding more in the future